docs/build/getting-started/fcl-quickstart.md

```diff
@@ -5,157 +5,502 @@ sidebar_label: Simple Frontend
+This command installs FCL and adds it to your project's dependencies.
 
-For our example, we are going to point at a free Access Node provided by Flow. Add the following config code to your `src/App.js`:
+Next, we'll configure FCL to connect to the [Flow Testnet](../../networks/flow-networks/accessing-testnet.md). An [Access Node](../../networks/node-ops/access-onchain-data/access-nodes/accessing-data/access-api.md) serves as the primary point of interaction for clients to communicate with the Flow network. It provides a gateway for submitting transactions, querying data, and retrieving information.
 
-```tsx
+In `src/App.js`, import FCL and add the configuration code:
+
+```jsx
 // src/App.js
 
 import * as fcl from '@onflow/fcl';
 
 fcl.config({
-  'accessNode.api': 'https://rest-testnet.onflow.org'
+  'accessNode.api': 'https://rest-testnet.onflow.org',
+});
+```
+
+This configuration sets the access node endpoint to the Flow Testnet.
+
+Your `src/App.js` should now look like this:
+
+```jsx
+import './App.css';
+import * as fcl from '@onflow/fcl';
+
+fcl.config({
+  'accessNode.api': 'https://rest-testnet.onflow.org',
 });
+
+function App() {
+  return (
+    <div className="App">
+      <div>FCL App Quickstart</div>
+    </div>
+  );
+}
+
+export default App;
 ```
 
 ## Querying the Chain
 
-On Flow, you can interact with a contract by reading from the chain with a script or changing its state with a transaction. Reading is free and is done with FCL by passing a [Cadence] script to `fcl.query`.
+Now, let's read data from a smart contract deployed on the Flow Testnet. We'll use a `HelloWorld` contract deployed to the account `0xa1296b1e2e90ca5b` (you can [view the contract here] to see what it looks like).
+
+This contract has a public variable `greeting` that we can read.
+
+First, we'll set up state in our app to store the greeting and manage component updates. We'll use React's `useState` and `useEffect` hooks.
 
-For our example we are going to read from a `HelloWorld` contract deployed to the account `0xa1296b1e2e90ca5b` on `testnet` (you can [view the contract here] to see what it looks like).
+Update your imports in `src/App.js` to include `useState` and `useEffect`:
 
-In the same `src/App.js` file, let's create app state to store our greeting and query the chain when the component renders in order to fetch the greeting state from the `HelloWorld` contract.
+```jsx
+import { useEffect, useState } from 'react';
+```
+
+Next, initialize the `greeting` state variable inside your `App` component:
+
+```jsx
+const [greeting, setGreeting] = useState('');
+```
+
+Now, let's create a function to query the greeting from the blockchain:
+
+```jsx
+const queryGreeting = async () => {
+  const res = await fcl.query({
+    cadence: `
+      import HelloWorld from 0xa1296b1e2e90ca5b
+
+      pub fun main(): String {
```

Comment:
@chasefleming I think this is the old way of doing this.  Shouldn't it be access(all)?

I'm getting:

 HTTP Request Error: An error occurred when interacting with the Access API.
      error=Invalid Flow argument: failed to execute script: [Error Code: 1101] failed to execute script at block (7ab1a2d983669a1aa3e9bade1270de00316ad98477cae5093e57ca741b170562): [Error Code: 1101] error caused by: 1 error occurred:
	* [Error Code: 1101] cadence runtime error: Execution failed:
error: `pub` is no longer a valid access keyword
 --> de8a43fd478ffdd20eae82630857d7bdd680c13673bfa37bc41a859a32d654a8:4:8
  |
4 |         pub fun main(): String {
  |         ^^^